test-watch.py

```diff
@@ -7,7 +7,7 @@ def main():
     except FileNotFoundError:
         pass
 
-    os.system('ptw -- --testmon')
+    os.system('ptw -- --testmon -vv --maxfail=1')
```

Comment:
The test driven development was being slowed down by having the tests continue after one was found to fail. This helped the process, and also gave more verbose debugging printouts.